### PR TITLE
Add arrayRemove wrapper for PostgreSQL function

### PR DESCRIPTION
**Pull request by sestrella**
Add support for `array_remove` function.

https://www.postgresql.org/docs/10/static/functions-array.html

### CODE REVIEW

**Comment by bitemyapp** [comment]
Looks fine in principle, need to resolve the merge conflict (that's you) and figure out why the test failed (either of us, I'll be taking a look as it wasn't obvious why it failed)

**Comment by sestrella** [comment]
@bitemyapp yes I also wasn't sure about what the build fail after taking a look at the log in travis since `stack test` output code was 0. I'm going to solve the conflicts right away.

**Comment by bitemyapp** [comment]
Think I found the source of the error: 

screenshot from 2018-01-10 12-20-05

**Comment by sestrella** [comment]
@bitemyapp thank you I'm going to take a look at that.

File: .travis.yml [hunk 1]
```diff
@@ -13,6 +13,8 @@ addons:
       - postgresql-client
       - postgresql-server-dev-all
 
+  postgresql: "9.6"
```
Comment (sestrella) [inline .travis.yml]: @bitemyapp I bumped PostgreSQL version. Hopefully, that solves the issue since I tested locally using the same version `9.6`.

**Comment by sestrella** [comment]
@bitemyapp bumping PostgreSQL did the trick. Thanks for your help.

**Comment by bitemyapp** [comment]
@sestrella thanks for the contributions and the follow-up!

**Pull request merged**